src/lib/numbers.ts

```diff
@@ -331,6 +331,18 @@ export function roundToTwoDecimals(n) {
   return Math.round(n * 100) / 100;
 }
 
+export function addTwoBigNumbers(a, b) {
```

Comment:
what if we will need to add more bigNumbers?
```
sumBigNumbers(...args) {
 return args.reduce(acc, value => acc.add(value || 0), BigNumber.from(0))
}
```

Reply:
Great idea!